content/publication/maranghi-2022-ai/index.md

```diff
@@ -41,7 +41,7 @@ projects: []
 publishDate: '2023-02-06T11:14:04.668002Z'
 publication_types:
 - '3'
-abstract: 'The Associazione Medici Diabetologi (AMD) col- lects and manages one of the largest worldwide- available collections of diabetic patient records, also known as the AMD database. This paper presents the initial results of an ongoing project whose focus is the application of Artificial Intel- ligence and Machine Learning techniques for con- ceptualizing, cleaning, and analyzing such an im- portant and valuable dataset, with the goal of pro- viding predictive insights to better support diabetol- ogists in their diagnostic and therapeutic choices.'
+abstract: 'The Associazione Medici Diabetologi (AMD) collects and manages one of the largest worldwide available collections of diabetic patient records, also known as the AMD database. This paper presents the initial results of an ongoing project whose focus is the application of Artificial Intelligence and Machine Learning techniques for conceptualizing, cleaning, and analyzing such an important and valuable dataset, with the goal of providing predictive insights to better support diabetol- ogists in their diagnostic and therapeutic choices.'
```

Comment:
diabel- ogists
██████████▀▀▀▀▀▀▀▀▀▀▀▀▀██████████ █████▀▀░░░░░░░░░░░░░░░░░░░▀▀█████ ███▀░░░░░░░░░░░░░░░░░░░░░░░░░▀███ ██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░██ █░░░░░░▄▄▄▄▄▄░░░░░░░░▄▄▄▄▄▄░░░░░█ █░░░▄██▀░░░▀██░░░░░░██▀░░░▀██▄░░█ █░░░██▄░░▀░░▄█░░░░░░█▄░░▀░░▄██░░█ ██░░░▀▀█▄▄▄██░░░██░░░██▄▄▄█▀▀░░██ ███░░░░░░▄▄▀░░░████░░░▀▄▄░░░░░███ ██░░░░░█▄░░░░░░▀▀▀▀░░░░░░░█▄░░░██ ██░░░▀▀█░█▀▄▄▄▄▄▄▄▄▄▄▄▄▄▀██▀▀░░██ ███░░░░░▀█▄░░█░░█░░░█░░█▄▀░░░░███ ████▄░░░░░░▀▀█▄▄█▄▄▄█▄▀▀░░░░▄████ ███████▄▄▄▄░░░░░░░░░░░░▄▄▄███████

Reply:
not my fault if they are called like that